p2p/proto/receipt.proto

```diff
@@ -42,6 +44,7 @@ message Receipt {
     repeated           MessageToL1 messages_sent = 3;
     ExecutionResources execution_resources       = 4;
     string             revert_reason             = 5;
+    Events             events                    = 6;
```

Comment:
```suggestion
    optional MessageToL2 consumed_message        = 6;
```

Reply:
Same

Reply:
Please view this [PR](https://github.com/starknet-io/starknet-p2p-specs/pull/17)